core/chainmgr/chainmgr.go

```diff
@@ -77,6 +76,7 @@ type ChainManager struct {
 	lock       sync.RWMutex
 	shards     map[common.ShardID]*ShardInfo
 	mainLedger *ledger.Ledger
+	ledger     *ledger.Ledger
```

Comment:
It's better to remove both mainLedger and ledger from chainmgr.
Getting ledger with ledger.GetShardLedger(...)?

Reply:
ok